problems/objects.js

```diff
@@ -141,7 +179,14 @@ const addsCountry = () => {};
  * @returns {Object} {"Mark Twain": 8.9, "Nathaniel Hawthorne": 5.1}
  */
 
-const authorScores = () => {};
+const authorScores = (authors) => {
+  let authorsObj ={}
+  for(let i = 0; i<authors.length; i++){
```

Comment:
Try using code formatters like prettier, so that the syntax are formatted the same throughout the entire project